src/AppRoot.tsx

```diff
@@ -673,7 +675,19 @@ class AppRoot extends React.Component<Props, State> {
   }
 
   public unconfigure = () => {
-    this.setState(this.initialState)
+    // Preserve hardware state after unconfigure.
+    /* eslint-disable @typescript-eslint/no-unused-vars */
+    const {
+      hasAccessibleControllerAttached,
+      hasCardReaderAttached,
+      hasChargerAttached,
+      hasPrinterAttached,
+      hasLowBattery,
+      ...rest
+    } = this.initialState
+    /* eslint-enable @typescript-eslint/no-unused-vars */
```

Comment:
We should replace this with something that is persistent and lives outside of the component tree.

Reply:
Created #1152.